<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>865</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="main_tab">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="text_tab">
      <attribute name="title">
       <string>Text</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7" rowstretch="0,0,0,0">
       <item row="3" column="0" alignment="Qt::AlignTop">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_12">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="3" column="0">
           <widget class="QGroupBox" name="text_sync_box">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
              <underline>false</underline>
             </font>
            </property>
            <property name="title">
             <string>Sync</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_11">
             <item row="0" column="0">
              <widget class="QLabel" name="text_mode_label">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                 <underline>false</underline>
                </font>
               </property>
               <property name="text">
                <string>Mode: </string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="text_input_label">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                 <underline>false</underline>
                </font>
               </property>
               <property name="text">
                <string>Input: </string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="text_led_box">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
              <underline>false</underline>
             </font>
            </property>
            <property name="title">
             <string>LED</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_10">
             <item row="1" column="2">
              <widget class="QLabel" name="text_pwm_label">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                 <underline>false</underline>
                </font>
               </property>
               <property name="text">
                <string>PWM: </string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="text_current_label">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                 <underline>false</underline>
                </font>
               </property>
               <property name="text">
                <string>Current: </string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="text_channel_label">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                 <underline>false</underline>
                </font>
               </property>
               <property name="text">
                <string>Channel:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="text_channel_name_label">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                 <underline>false</underline>
                </font>
               </property>
               <property name="text">
                <string>Channel Name:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QGroupBox" name="text_temperature_box">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
              <underline>false</underline>
             </font>
            </property>
            <property name="title">
             <string>Temperature</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QLabel" name="text_transistor_label">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                 <underline>false</underline>
                </font>
               </property>
               <property name="text">
                <string>Transistor: </string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="text_driver_fan_label">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                 <underline>false</underline>
                </font>
               </property>
               <property name="text">
                <string>Driver Fan: </string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="text_resistor_label">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                 <underline>false</underline>
                </font>
               </property>
               <property name="text">
                <string>Resistor: </string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="text_external_label">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                 <underline>false</underline>
                </font>
               </property>
               <property name="text">
                <string>External: </string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="text_external_fan_label">
               <property name="text">
                <string>External Fan: </string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="text_connection_box">
            <property name="font">
             <font>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="title">
             <string>Connection</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="0" column="1">
              <widget class="QLabel" name="text_serial_label">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                 <underline>false</underline>
                 <strikeout>false</strikeout>
                </font>
               </property>
               <property name="text">
                <string>Serial: </string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="text_name_label">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                 <underline>false</underline>
                 <strikeout>false</strikeout>
                </font>
               </property>
               <property name="text">
                <string>Name: </string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="text_com_port_label">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                 <underline>false</underline>
                 <strikeout>false</strikeout>
                </font>
               </property>
               <property name="text">
                <string>COM Port: </string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="text_control_label">
               <property name="text">
                <string>Control:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="graph_tab">
      <attribute name="title">
       <string>Graph</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,25,25,40,10">
       <item row="6" column="0">
        <widget class="QGroupBox" name="graph_temperature_external_box">
         <property name="title">
          <string>Temperature: External</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="PlotWidget" name="graph_temperature_external"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QGroupBox" name="graph_temperature_internal_box">
         <property name="title">
          <string>Temperature: LED Driver</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="PlotWidget" name="graph_temperature_internal"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QGroupBox" name="graph_current_box">
         <property name="title">
          <string>LED Intensity:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="PlotWidget" name="graph_current"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QGroupBox" name="graph_update_box">
         <property name="title">
          <string>Update Speed</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6" columnstretch="20,20,20,0,0,0">
          <item row="0" column="5">
           <widget class="QLabel" name="graph_custom_update_label">
            <property name="text">
             <string>Hz</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QDoubleSpinBox" name="graph_custom_update_spinbox">
            <property name="minimum">
             <double>0.010000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QRadioButton" name="graph_slow_update_button">
            <property name="text">
             <string>0.2 Hz</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">update_speed_button_group</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="graph_fast_update_button">
            <property name="text">
             <string>5 Hz</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">update_speed_button_group</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="graph_normal_update_button">
            <property name="text">
             <string>1 Hz</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">update_speed_button_group</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QRadioButton" name="graph_custom_update_button">
            <property name="text">
             <string>Custom:</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">update_speed_button_group</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="update_speed_button_group"/>
 </buttongroups>
</ui>
